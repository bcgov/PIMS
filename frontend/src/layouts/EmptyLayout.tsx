import './PublicLayout.scss';

import { EmptyHeader, Footer } from 'components/layout';
import React from 'react';
import { Container } from 'react-bootstrap';

const EmptyLayout: React.FC = ({ children }) => {
  return (
    <>
      <Container fluid className="App">
        <header className="header-layout fixed-top">
          <Container className="px-0">
            <EmptyHeader />
          </Container>
        </header>

        <main className="App-content">{children}</main>

        <footer className="footer-layout fixed-bottom">
          <Container className="px-0">
            <Footer />
          </Container>
        </footer>
      </Container>
    </>
  );
};

export default EmptyLayout;
