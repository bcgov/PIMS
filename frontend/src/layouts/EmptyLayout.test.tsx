import EmptyLayout from './EmptyLayout';
import { render } from '@testing-library/react';
import React from 'react';

describe('Empty Layout', () => {
  it('renders', () => {
    const { container } = render(<EmptyLayout></EmptyLayout>);
    expect(container.firstChild).toMatchSnapshot();
  });
});
