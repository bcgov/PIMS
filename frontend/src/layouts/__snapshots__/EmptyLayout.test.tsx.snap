// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty Layout renders 1`] = `
.c0 {
  border-left: 2px solid white;
  font-size: 34px;
  margin: 0 15px 0 25px;
  vertical-align: top;
}

<div
  class="App container-fluid"
>
  <header
    class="header-layout fixed-top"
  >
    <div
      class="px-0 container"
    >
      <nav
        class="App-header navbar navbar-expand navbar-light"
      >
        <span
          class="brand-box navbar-brand"
        >
          <a
            href="https://www2.gov.bc.ca/gov/content/home"
            rel="noopener noreferrer"
            target="_blank"
          >
            <img
              alt="Government of BC logo"
              class="bc-gov-icon"
              height="43"
              src="() => /*#__PURE__*/(0, _jsxDevRuntime.jsxDEV)(\\"template\\", {
  children: /*#__PURE__*/(0, _jsxDevRuntime.jsxDEV)(\\"svg\\", {}, void 0, false, {
    fileName: _jsxFileName,
    lineNumber: 5,
    columnNumber: 5
  }, void 0)
}, void 0, false, {
  fileName: _jsxFileName,
  lineNumber: 4,
  columnNumber: 3
}, void 0)"
              width="156"
            />
          </a>
          <span
            class="c0"
          />
          <img
            alt="PIMS logo"
            class="pims-logo"
            height="50"
            src="logo_only.png"
          />
        </span>
        <div
          class="title mr-auto navbar-nav"
        >
          <div
            class="nav-item"
          >
            <h1
              class="longAppName"
            >
              Property Inventory Management System
            </h1>
            <h1
              class="shortAppName"
            >
              PIMS
            </h1>
          </div>
        </div>
      </nav>
    </div>
  </header>
  <main
    class="App-content"
  />
  <footer
    class="footer-layout fixed-bottom"
  >
    <div
      class="px-0 container"
    >
      <nav
        class="footer navbar navbar-expand navbar-light"
      >
        <a
          class="nav-link"
          href="http://www.gov.bc.ca/gov/content/home/disclaimer"
        >
          Disclaimer
        </a>
        <a
          class="nav-link"
          href="http://www.gov.bc.ca/gov/content/home/privacy"
        >
          Privacy
        </a>
        <a
          class="nav-link"
          href="http://www.gov.bc.ca/gov/content/home/accessible-government"
        >
          Accessibility
        </a>
        <a
          class="nav-link"
          href="http://www.gov.bc.ca/gov/content/home/copyright"
        >
          Copyright
        </a>
      </nav>
    </div>
  </footer>
</div>
`;
